tests/Captioning/ConverterTest.php

```diff
@@ -16,23 +16,23 @@ public function testSubrip2WebvttConversion()
 
         $content = "WEBVTT
 
-00:00:00.000 --> 00:00:20.000
+00:00:01.000 --> 00:00:20.000
```

Comment:
Why did you change all the timestamps exactly?

No that it really matters, but I was just wondering.